docs/api/reading.md

```diff
@@ -172,7 +172,7 @@ GET /people?order=age.nullsfirst
 GET /people?order=age.desc.nullslast
 ```
 
-To filter the embedded items, you need to specify the tree path for the order param like so. 
+To order the embedded items, you need to specify the tree path for the order param like so.
 ```HTTP
 GET /projects?select=id,name,tasks{id,name}&order=id.asc&tasks.order=name.ask
```

Comment:
Oh looks like the `.ask` was a previous typo, want to include a fix here while you're at it?